packages/liveblocks-core/src/comments/index.ts

```diff
@@ -42,6 +42,16 @@ export type CommentsApi<ThreadMetadata extends BaseMetadata> = {
     threadId: string;
     commentId: string;
   }): Promise<void>;
+  addCommentReaction(options: {
```

Comment:
Not sure if these names are set in stone already? Would we rename those to `addReaction` and `removeReaction` maybe? A tad bit shorter, yet just as clear and still unambiguous.

Reply:
(And from the params it's already clear they need to be attached to a comment.)

Reply:
Not in stone! I was wondering about this exactly: https://github.com/liveblocks/liveblocks/pull/1155#:~:text=API%20feedback%3A%20addCommentReaction%20vs%20addReaction%3F. I'll wait for more eyes but I'm leaning towards `addReaction`/`removeReaction` as well.

Reply:
I also like `addReaction` and `removeReaction`!